### ISSUE TITLE
Erros de Digitação

### ISSUE THREAD

**Issue by claleni** [ud-portugues/POeTiSA#4]
Na seguinte frase ' "Tigresa", honra ou insulta as mulheres.', claramente está faltando uma crase para fazer sentido.
Eu coloco esse "as" como determinante ou ERRO_DE_DIGITACAO?

**Comment by claleni** [comment]
Tem como resgatar o contexto dessa frase? Porque eu não sei se está horando e insultando as mulheres ou se é uma honra ou insulta às mulheres.
E isso muda tudo na frase.

**Comment by MagaliDuran** [comment]
> Na seguinte frase ' "Tigresa", honra ou insulta as mulheres.', claramente está faltando uma crase para fazer sentido.
> Eu coloco esse "as" como determinante ou ERRO_DE_DIGITACAO?

Não tem erro de digitação no "a", mas creio que haja erro de digitação na palavra "insulta" pois, como está coordenada com o substantivo "honra", deveria ser o substantivo "insulto", (o qual prevê complemento nominal introduzido pela preposição "a"). 
Honrar e insultar são verbos transitivos diretos, não são regidos por preposição. Se encontrar a preposição "a", é com uso expletivo e não porque o verbo exige. Normalmente o uso expletivo da preposição "a" aparece quando o objeto direto é Deus: "honrar a Deus", assim como "amar a Deus".

**Comment by claleni** [comment]
Eu tinha entendido "honra" e "insulta" como verbos com uso expletivo. Como no caso de "honrar a Deus".
Mas eles também podem ser substantivos, como "honra ou insulto". Mas aí o "as" mulheres parece ficar sem nexo como DET.
Por isso perguntei se poderia resgatar o contexto, porque fiquei na dúvida onde está o erro e achei que era uma falta de crase.

**Comment by MagaliDuran** [comment]
Clarissa, desculpe-me, pois compliquei a resposta. Simplificando: como honra e insulta são verbos,  não tem crase,  pois são verbos transitivos diretos.

**Comment by gracanunes** [comment]
Magali, como devemos anotar erros do tipo: "tem" no lugar de "têm"?
Não anotar o erro? erro de digitação? 
Obrigada!

**Comment by MagaliDuran** [comment]
Os erros ortográficos serão considerados erros de digitação. Neles se enquadram erros propositais (goxto ao invés de gosto) e erros não propositais. Falta de acento,  letra faltante,  letra trocada,  tudo isso é erro ortográfico e, portanto,  erro de digitação.

**Comment by GaybsGimenez** [comment]
Oi!! nessa frase : "Cortes em o orçamento científico têm um impacto que os que seus autores não entendem" , esse segundo "que", é erro de digitação? pois parece que ele não faz parte dessa frase.

**Comment by MagaliDuran** [comment]
Sim, com certeza.

Em qua, 14 de abr de 2021 11:53, GaybsGimenez ***@***.***>
escreveu:

**Comment by claleni** [comment]
Olá, tenho algumas dúvidas quanto às correções do Pacote 10 enviadas por e-mail.
Vou tentar tirar algumas delas já, porque acho que não vai dar para tirar todas minhas dúvidas na próxima reunião.
Na seguinte sentença:

"Elisabeth Moss venceu **em em uma** de as categorias..."

O Erro de Digitação deve estar na primeira palavra **em** ou na segunda?
Eu entendi que seria na primeira palavra numa das reuniões que tivemos. E que quando tem vários erros de processamento, deve estar na palavra anterior a eles, certo?

No caso da sentença:
"Porém, após o alerta de emergência vindo de o alto volume de chuvas, as **doenças hídrica** também tornaram-se prioridade."

Por que **hídrica** não é E_DIGT?

**Comment by gracanunes** [comment]
> Olá, tenho algumas dúvidas quanto às correções do Pacote 10 enviadas por e-mail.
> Vou tentar tirar algumas delas já, porque acho que não vai dar para tirar todas minhas dúvidas na próxima reunião.
> Na seguinte sentença:
> 
> "Elisabeth Moss venceu **em em uma** de as categorias..."
> 
> O Erro de Digitação deve estar na primeira palavra **em** ou na segunda?
> Eu entendi que seria na primeira palavra numa das reuniões que tivemos. E que quando tem vários erros de processamento, deve estar na palavra anterior a eles, certo?
> 
> No caso da sentença:
> "Porém, após o alerta de emergência vindo de o alto volume de chuvas, as **doenças hídrica** também tornaram-se prioridade."
> 
> Por que **hídrica** não é E_DIGT?

Ola Clarissa,

a anotação dos erros de digitação e pré-processamento ainda está um pouco confusa, e em breve vamos tentar uniformizá-las. Mas tenham em mente duas coisas: 

1) anotar um desses erros serve para chamar a atenção da equipe (no caso, a Lucelene) para corrigir ou ignorar a tal sentença onde o erro ocorreu, portanto, a exata posição não é tão relevante assim.

2) em alguns casos - como esse, do plural - talvez não vale a pena "perder" a sentença por conta desse erro, afinal, ele não altera em nada a estrutura sintática ou mesmo o significado. Daí poderíamos ignorar esse tipo de erro.

Mas, repito, vamos tentar uniformizar essas decisões para que isso não impacte nos índices de concordância.

Abraços,
Graça

**Comment by MagaliDuran** [comment]
Todos os erros de concordância e de ortografia devem ser marcados como
E_DIGIT para que possamos consertar e tornar a sentença um exemplo da norma
culta.

Em sex, 2 de abr de 2021 12:16, gracanunes ***@***.***>
escreveu:

**Comment by MagaliDuran** [comment]
Sim, é E_DIGIT e bastará no pós-processamento suprimir o "que" para tornar
a sentença correta.

Em qua, 14 de abr de 2021 11:53, GaybsGimenez ***@***.***>
escreveu:
